backbone.queryparams.js

```diff
@@ -309,8 +312,14 @@ function iterateQueryString(queryString, callback) {
   var keyValues = queryString.split('&');
   _.each(keyValues, function(keyValue) {
     var arr = keyValue.split('=');
-    callback(arr.shift(), arr.join('='));
+    if (!containsInvalidKey(keyValue)) {
+      callback(arr.shift(), arr.join('='));
+    }
   });
 }
 
+function containsInvalidKey(key) {
```

Comment:
the name of the param `key` suggest its a single key, but its an array of keys, correct?